eng/GenAPI.exclusions.txt

```diff
@@ -0,0 +1,5 @@
+# Manually implemented - https://github.com/dotnet/arcade/pull/2033
```

Comment:
Can you open issues on our side to track all manually implemented ref assembly code? I want to make sure we follow up and don't accidentally ship refs which have drifted from implementation.